Client/Application/LeanplumIntegration.swift

```diff
@@ -281,11 +281,11 @@ class LeanPlumClient {
         return (prefs?.stringForKey(PrefsKeys.KeyMailToOption) ?? "mailto:") == "mailto:"
     }
 
-    private func getSettings() -> LPSettings? {
+    func getSettings() -> LPSettings? {
         let bundle = Bundle.main
-        guard let appId = bundle.object(forInfoDictionaryKey: LPAppIdKey) as? String,
-              let productionKey = bundle.object(forInfoDictionaryKey: LPProductionKeyKey) as? String,
-              let developmentKey = bundle.object(forInfoDictionaryKey: LPDevelopmentKeyKey) as? String else {
+        guard let appId = bundle.object(forInfoDictionaryKey: LPAppIdKey) as? String, !appId.isEmpty,
```

Comment:
Bundle produced empty string even if it had no value in the plist file.
Hence the additional check for isEmpty